seed_helen_comments.py

```diff
@@ -0,0 +1,354 @@
+    # buckets the colors to convert colors from rgb to css friendly names
+
+    min_colors = {}
+
+    # loops over css3 named hex/color items (138 pairs)
+    # next, pull r, g, b, values from image color tuple
+    # then convert hex to rgb with just key (hex values)
+
+    for key, name in css3_hex_to_names.items():
+        r_c, g_c, b_c = hex_to_rgb(key)
+        # an example might be 124 125 0
+        # so r = (124 - color[0]) ^ 2
+        # color is the inputted rgb value tuples
+
+        r = (r_c - color[0]) ** 2
+        g = (g_c - color[1]) ** 2
+        b = (b_c - color[2]) ** 2
+
+        min_colors[(r + g + b)] = name
+        # print "MIN COLORS", min_colors
+        # print "MIN KEYS ONLY", min_colors[min(min_colors.keys())]
+    return min_colors[min(min_colors.keys())]
+
+
+# def show_colors(final_colors):
+#     # A FXN TO AGGREGATE TOP COLORS BY SHOW
+#     # final_colors is PER image, need to aggregate all final_colors to show colors
+#     show_colors = []
+#     for color in final_colors:
+#         show_colors.append(color)
+
+#     show_colors = final_colors[:6]
+#     return show_colors
+
+
+def get_color_name(color):
+    # pillow errors out if it doesnt have the exact color name to rgb
+
+    # print "at get_color_name"
+    try:
+        closest_name = actual_name = rgb_to_name(color)
+    except ValueError:
+        closest_name = closest_color(color)
+        actual_name = None
+    return closest_name
+
+    # actual_name, closest_name = get_color_name(color)
+
+    # print "closest color name: ", closest_name
+
+
+def get_colors(img, img2):
+    # this function takes in bg & foreground colors and returns top img colors
+    # 1. convert image to palette format
+    img = img.convert('P')
+    img2 = img2.convert('P')
+
+    # 2. convert imgs to RGB for getcolors fxn - returns rgb tuples
+    bg_colors = (img2.convert('RGB').getcolors())
+    foreground_colors = (img.convert('RGB').getcolors())
+
+    # 3. set up our lists to capture top colors for bg & foreground
+    final_bg_colors = []
+    final_colors = []
+    # 4. set up empty list to capture named final colors
+    final_named_colors = []
+
+    named_bg_colors = []
+    named_foreground_colors = []
+
+    # 6. these two loops return lists of named colors
+    for count, rgb in bg_colors:
+        named_bg_colors.append(get_color_name(rgb))
+    for count, rgb in foreground_colors:
+        named_foreground_colors.append(get_color_name(rgb))
+
+    # 7. these create color/count pairs 
+    count_bg_colors = {color: named_bg_colors.count(color) for color in named_bg_colors}
+    count_foreground_colors = {color: named_foreground_colors.count(color) for color in named_foreground_colors}
+
+    # 8. these take the color/count pairs and sort them
+    sorted_bg_colors = sorted(count_foreground_colors.items(), key=lambda x: x[1], reverse=True)[:10]
+    sorted_foreground_colors = sorted(count_foreground_colors.items(), key=lambda x: x[1], reverse=True)[:10]
```

Comment:
It looks like from the beginning of this function to here, you are doing the exact same thing two times with two different variables.  Maybe you should DRY this up (Do not Repeat Yourself).  Imagine if you wanted to change something seemingly small-- you would have to change it multiple times.
Maybe something like
`sorted_bg_colors = get_sorted_colors(img)`
`sorted_foreground_colors = get_sorted_colors(img2)`
`....`